src/metrics/activity_openrank.py

```diff
@@ -0,0 +1,239 @@
+from basic import QueryConfig, \
+                  getMergedConfig, \
+                  getRepoWhereClauseForNeo4j, \
+                  getTimeRangeWhereClauseForNeo4j, \
+                  getTimeRangeSumClauseForNeo4j, \
+                  getUserWhereClauseForNeo4j, \
+                  getRepoWhereClauseForClickhouse,\
+                  getUserWhereClauseForClickhouse,\
+                  getTimeRangeWhereClauseForClickhouse,\
+                  getGroupArrayInsertAtClauseForClickhouse,\
+                  getGroupTimeAndIdClauseForClickhouse
+import db.neo4j_driver as neo4j_driver
+from label_data_utils import getLabelData
+import db.clickhouse as clickhouse
+from functools import cmp_to_key
+import numpy as np
+
+ISSUE_COMMENT_WEIGHT = 1
+OPEN_ISSUE_WEIGHT = 2
+OPEN_PULL_WEIGHT = 3
+REVIEW_COMMENT_WEIGHT = 4
+PULL_MERGED_WEIGHT = 2
+
+def get_RepoActivityOrOpenrank(config, type='activity'):
```

Comment:
We should keep an identical function style, underscore or camel. But not both at same time. Did you do this for a particular reason?

Reply:
> We should keep an identical function style, underscore or camel. But not both at same time. Did you do this for a particular reason?

sorry, I will check all functions.